lib/Controller/RoomController.php

```diff
@@ -219,6 +219,15 @@ protected function formatRoom(Room $room, Participant $participant = null) {
 			}
 		}
 
+		$inactiveUsers = array_filter($participants['users'], function($data) {
+			return $data['lastPing'] <= time() - 30 &&
```

Comment:
This is a bit drastic? If one ping lags, you get killed out of the room.
The user tries 3 (or5) reconnects.

Reply:
I just used the same timeout already being used for guests. Should I increase the timeout for both? Only for registered users? Or leave it like it is now?

Reply:
Please use 100 seconds, so we are in 3 pings a 30 seconds + buffer
https://github.com/nextcloud/spreed/blob/db5861e3902bb895e874f911e9e8d2c43e28682a/js/signaling.js#L660-L664